app/_containers/PostCard/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import styles from "./index.module.scss";
+import Image from "next/image";
+
+interface Post {
+    id: number
+    title: string
+    description: string
+    imageUrl: string
+    tags: string[]
+  }
+
+export default function PostCard ({ post }: { post: Post }){
+  return (
+    <article className={styles.article}>
+      {/* Card image */}
+      <div className={styles.cardImg}>
+        <img 
```

Comment:
Please use next image for optimization.

https://nextjs.org/docs/pages/api-reference/components/image